tor_core/initialize.py

```diff
@@ -184,6 +186,18 @@ def populate_subreddit_lists(config):
         )
     )
 
+    config.subreddits_domain_filter_bypass = get_wiki_page(
+        'subreddits/domain-filter-bypass', config
+    ).split('\r\n')
```

Comment:
Not really a problem as it is now, but you can also use `splitlines` which will work regardless of newline type.